prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      var fullName = 'James Cody Crawford';
+      console.log(fullName);
+
+      var numberOfPets = 'Two';
```

Comment:
Here you should leave out the quotation marks and use a literal `2` so that JavaScript knows that `numberOfPets` should be a number and not a string.